modules/ROOT/pages/api-led-deploy.adoc

```diff
@@ -45,111 +47,96 @@ GET http://hello-world-greeting.sandy-kim.us-e2.cloudhub.io/api/greeting
-
 Now you have an implementation URL that you don't want to share with the world, and a proxy URL that is safe to share with the world after you configure it.
 
-image:proxy-deploy2.png[settings page showing URLs, 600]
-
 To configure the proxy:
 
-. In the *Proxy* section, right-click the link after *Proxy URL* and copy the link.
-. At the top of the page, click *Add consumer endpoint*.
-. Paste in the link you just copied, and press Return to save the link.
+. In *Anypoint Manager > API Administration*, click the *hello-world-api* link.
+. In *API Summary*, right-click the *Proxy URL* link and copy it. 
+. Click the pencil icon to edit the API instance. 
+. In the *Downstream* section, paste in the link you just copied in to *Consumer endpoint*, and then click *Save & Apply*.
 
 Now you can test the proxy endpoint from Exchange:
 
-. Click *View API in Exchange*.
-+
-image:exchange-page-for-proxy.png[Exchange page for hello-world proxy, 600]
-
-. Click *API instances*. You can see the Mocking Service version and the new proxy you just created in the Sandbox environment, labeled "no policy."
+. Select *Actions > View API in Exchange*.
+. Click the Expand icon, then click *Manage versions.* You can see the Mocking Service version and the new proxy you just created in the Sandbox environment.
+. Click *Close*.
 . Click */greeting* to open, then click *GET*.
-. Click *Mocking Service* to open the drop-down menu.
-. Select *Sandbox: No policy instance*.
+. In *Select server*, expand the menu and select *Sandbox - v1:<version>*.
++
+image:api-led-select-server-exchange.png[Expanded more menu to show the list of available servers]
++
 . Click *Send*. The response should be `200 OK` with the greeting `Today the greeting is Hello.`
 
 Finish configuring the proxy:
 
-. Click *API instances* to return to the list of API instances in Exchange.
-. Select your API.
-. Click *Private* and then choose *Public*.
+. Click *Home* to return to the API summary.
+. Click *Shar*.
+. In the *Share* dialog window, click *Public* and then select the API version to make available on the public Exchange portal.
++
+image:api-led-share-api.png[Share window with Public and the version v1 checkbox selected]
++
+. Click *Save*.
 
 == Step 5.4: Add a Rate Limit Policy
 
 To add a rate limit policy:
 
 . Return to API Manager.
-. Click *hello-world*.
-. Click *v1*.
-. In the navigation, select *Policies*.
-. Click *Apply New Policy*.
-. Don't change the default *All Mule Versions*.
-. Scroll to locate *Rate limiting* (not *Rate limiting - SLA based*).
-. Click *Rate limiting*, select *1.3.5* or a later version if available, and click *Configure Policy* at the bottom of the dialog box.
-. In the Apply Rate limiting policy page, set the limit values:
-  ** *# of Reqs*: `3`
+. In *API Administration*, click the *hello-world-api* link.
+. In the navigation, click *Policies*.
+. Click *+ Add policy*.
```

Comment:
I think our guidance is to not include punctuation in UI elements (buttons, menus), but maybe that's terminating punctuation. 🤔

Reply:
No, I think you're right.